odoo/addons/base/models/ir_actions.py

```diff
@@ -272,6 +272,9 @@ def _check_view_mode(self):
     target = fields.Selection([('current', 'Current Window'), ('new', 'New Window'), ('inline', 'Inline Edit'), ('fullscreen', 'Full Screen'), ('main', 'Main action of Current Window')], default="current", string='Target Window')
     view_mode = fields.Char(required=True, default='tree,form',
                             help="Comma-separated list of allowed view modes, such as 'form', 'tree', 'calendar', etc. (Default: tree,form)")
+    mobile_view_mode = fields.Char(default="kanban", help="View mode override in mobile and small "
+                                             "screen environments. If unset, will default to kanban view "
+                                             "if available, otherwise the same view as for desktop mode")
```

Comment:
```suggestion
    mobile_view_mode = fields.Char(default="kanban", help="First view mode in mobile and small screen environments (default='kanban'). If it can't be found among available view modes, the same mode as for wider screens is used)"
```